packages/@coorpacademy-components/src/template/my-learning/index.js

```diff
@@ -471,7 +469,7 @@ const MyLearning = (props, context) => {
           </div>
         ) : (
           <div className={style.skillListContainer}>
-            {activeFilter === 'review_mode_available' && filters[activeFilter].length === 0 ? (
+            {activeFilter === 'review' && filters[activeFilter].length === 0 ? (
```

Comment:
pourquoi ça a changé ça?

Reply:
actuellement sur master de components quand on clique sur available for review et qu'il n'y a rien, il n'y a pas le empty state : https://coorpacademy.github.io/components/components/?path=/story/template-mylearning--no-review-skill
J'ai vu qu'il cherche le nom du filters et qu'en terme de noms il y a 2 choix. C'est une correction.
`  const filters = useMemo(() => {
    return {
      all: searchResults,
      review: skillsReviewReady
    };
  }, [searchResults, skillsReviewReady]);`